components-core/src/main/java/com/airwallex/android/core/AirwallexSession.kt

```diff
@@ -50,4 +50,9 @@ abstract class AirwallexSession : Parcelable {
      * Google Pay options
      */
     abstract val googlePayOptions: GooglePayOptions?
+
+    /**
+     * limit the payment methods displayed on the list screen
```

Comment:
Shall we refer the user to the API reference `https://www.airwallex.com/docs/api#/Payment_Acceptance/Config/_api_v1_pa_config_payment_method_types/get`, so that they will name this after `name` of `items` in the response.

Reply:
I will add the API reference to the comment